### DISCUSSION TITLE
video goes black by pause

### DISCUSSION THREAD

**Discussion by somboku** [smplayer-dev/smplayer#291] [General]
Guys, 

first but first. this one is great. thanks for doing that effort. 

Now, when I press pause the video gets black but "sadly" not always but to 98%. 
Did I missed something in the settings? 
I use kernel 5.13.0-21 and integrated Intel Graphic Chip. 

thanks. 
S.

**Comment by smplayer-dev** [comment]
Try enabling or disabling the option  "repaint the background of the video window".
